soc/riscv/riscv-privilege/telink_b9x/Kconfig.soc

```diff
@@ -38,7 +38,7 @@ config TELINK_B9x_HWDSP
 config TELINK_B9X_MBEDTLS_HW_ACCELERATION
 	bool "Use Telink B9x platform mbedtls HW acceleration"
 	depends on SOC_SERIES_RISCV_TELINK_B9X && MBEDTLS
-	default y
+	default n
```

Comment:
This will disable HW crypto for all B9X SoCs
Revert this change and if you need to disable only for B95, jut set "CONFIG_TELINK_B9X_MBEDTLS_HW_ACCELERATION=n" in some conf or Kconfig file of B95 SoC (board)

Reply:
TELINK_B9X_MBEDTLS_HW_ACCELERATION=y required for Matter
there two cases:
1. Disable only for B95 and will need to rework Matter. Anyway will need in near future.
2. Revert to Y and create jira ticket for B95 to Adopt/fix and test HW crypto for B95. Better to do this.

Reply:
already revert in Kconfig.soc , and set the TELINK_B9X_MBEDTLS_HW_ACCELERATION=y in the tlsr9258a_defconfig

Reply:
About CONFIG_TELINK_B9X_MBEDTLS_HW_ACCELERATION, I have create a ticket ZEPHYR-503, see my comment in it.